docs/workflow.md

```diff
@@ -31,3 +31,24 @@ Deploy each PR separately and in ascending order.
 - PR #2: Generate the migration
 
 Deploy each PR separately and in ascending order.
+
+## Continuous Integration and Review Apps
+
+Opening a PR activates different services:
```

Comment:
Would it make sense to add links to "our spaces" for each of these services? Or would that be helping people who want to gain unauthorized access too much.

Reply:
It's not useful in this context: links are available per PR.